faststream/nats/message.py

```diff
@@ -17,6 +17,11 @@ async def ack(self) -> None:
             await self.raw_message.ack()
         await super().ack()
 
+    def ack_sync(self) -> None:
+        if not self.raw_message._ackd:
+            self.raw_message.ack()
```

Comment:
Can you please call `raw_message.ack_sync()` here?

Reply:
Sure, will update.